### DISCUSSION TITLE
Desugaring error

### DISCUSSION THREAD

**Discussion by patryk84a** [shreyashsaitwal/rush-cli#51] [General]
I get this error when desugaring, from ExoPlayer2 library 2.16.0 onwards. 2.15.1 compiles fine. What could this be and what can I try?

`Exception in thread "main" java.lang.Verify class com.google.android.exoplayer2.ExoPlayerImpl overrides final method stop.()V`

**Comment by shreyashsaitwal** [comment]
Do you get this while building the extension or during the app's runtime?

**Reply by patryk84a** [reply]
Hi.  Thanks for the answer.  When building an extension, more precisely - during desugaring.  ExoPlayer2 library from version 2.16.0 onwards.  I can't find the reason, google is also silent about it.

I thought it was something with the stop() method but I removed the stop from my extension but it didn't help.

**Reply by shreyashsaitwal** [reply]
It's hard to say anything precise here since this is more of an issue from the desugaring tool's side. I will try using the said library myself and see if I can reproduce and fix the issue.

Edit: @patryk84a, can you link me to the library?

**Reply by patryk84a** [reply]
I have some progress. I was able to compile an almost empty project with this library. So I suspect something in my code is the problem. Perhaps whose methods are not compatible between libraries.

**Reply by patryk84a** [reply]
I managed to build the extension on the library version 2.16.1. Now trying 2.17.1. I get an error that a certain class is missing, but this class is. 
Can I send you my zipped sample project via PM in the AppInventor community?

**Reply by shreyashsaitwal** [reply]
> Can I send you my zipped sample project via PM in the AppInventor community?

Sure! That'd be a lot better.
